### ISSUE TITLE
Consider reaching out to developers and/or add information about the fetch errors page to documentation

### ISSUE THREAD

**Issue by seankmartin** [napari/npe2api#18]
Hi all,

I wanted to bring up a couple of points regarding the [fetch errors page](https://npe2api.vercel.app/errors.json) in the npe2api:

**Documentation for plugin developers**: I came across the [errors.json](https://npe2api.vercel.app/errors.json) page recently, which contains information about fetch errors related to plugins. It got me thinking whether we should document this page and make plugin developers aware of it. By doing so, we can inform them that if their plugin appears on this page, users might encounter issues while installing it.

**Reaching out to developers**: I was also wondering if it could be good to reach out to the developers of the plugins listed in [errors.json](https://npe2api.vercel.app/errors.json). These developers might not be aware that their plugins are causing fetch errors, and it's likely that users will face difficulties installing these plugins.

I encountered a similar situation when I faced installation issues with a plugin, and it turned out to be listed on the fetch errors page. It made me realise that the problem was not specific to my local setup but rather a widespread issue with the plugin. I think it may be beneficial to developers if they knew about this page to help avoid this kind of situation.

**Comment by tlambert03** [comment]
For what it's worth, I did reach out to everyone in the errors at one point (probably a year ago). So some of the ones on there were notified... but lots of stuff has changed I'm sure.  I think documenting it is a good idea (just a small little side box somewhere).

**Comment by DragaDoncila** [comment]
@willingc agreed, at a minimum we should link to the json in our troubleshooting guide and in our `npe2 adaptor` doc
